### ISSUE TITLE
[feature request] word meanings from google in one field

### ISSUE THREAD

**Issue by omega3** [1goodday/Google-Dictionary.ankiaddon#2]
I would like to have all word meanings in one line separated by slash, like this (output is in Polish):

zakończyć / wnioskować / wywnioskować / sfinalizować / konkludować / dokończyć / finalizować / zakończyć /

In Linux terminal `trans` gives me the output:

```
trans conclude
conclude
/kənˈklo͞od/

wyciągnąć wniosek

conclude – definicje
[ English -> Polski ]

czasownik
    zakończyć
        end, complete, finish, terminate, conclude, wind up
    wnioskować
        conclude, reason, gather, argue, collect
    wywnioskować
        deduce, conclude, infer, understand, gather, induce
    sfinalizować
        finalize, conclude, sign, settle, button up
    konkludować
        conclude, infer
    dokończyć
        complete, finish, end, finish up, conclude, do away
    finalizować
        conclude, settle

conclude
    wyciągnąć wniosek, wnioskować, zakończyć, wywnioskować, sfinalizować
```

Even this output:
`wyciągnąć wniosek, wnioskować, zakończyć, wywnioskować, sfinalizować`
would be enough.

I would like to have it in a field like this (red letters):
https://i.imgur.com/hsZvTOX.png

As reference, maybe this might be useful:
translate-shell - can do something similar, I mean create such list, so maybe some part of its code maybe useful
https://github.com/soimort/translate-shell/issues/368

Basically the author here is right, so if only possible to have / or | instead of comma.

If not possible to implement it in this add-on maybe it would be possible for you to make it in separate add-on.

This listing of all meaning helps me to understand an idea behind a word. So it is super-useful when learning a language.
